Lib/test/test_asyncio/test_events.py

```diff
@@ -2235,7 +2235,7 @@ def sendfile_native(transp, file, offset, count):
                                     "not supported"):
             self.run_loop(
                 self.loop.sendfile(cli_proto.transport, self.file,
-                                   fallback=False))
+                                  fallback=False))
```

Comment:
This looks like an unrelated change that will cause a linting error, no?

Reply:
fixed